website/docs/d/cloud_provider_access.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: mongodbatlas_cloud_provider_access"
+sidebar_current: "docs-mongodbatlas-datasource-cloud-provider-access"
+description: |-
+    Allows you to get the list of cloud provider access roles
+---
+
+# mongodbatlas_cloud_provider_access
+
+`mongodbatlas_cloud_provider_access` allows you to get the list of cloud provider access roles, by now AWS is supported.
```

Comment:
Slight edit:  `mongodbatlas_cloud_provider_access` allows you to get the list of cloud provider access roles, currently only AWS is supported.